src/appkit.cs

```diff
@@ -2619,7 +2619,7 @@ public interface NSCollectionView : NSDraggingSource, NSDraggingDestination {
 		//NSImage DraggingImage (NSIndexSet itemIndexes, NSEvent evt, NSPointPointer dragImageOffset);
 
 		//Detected properties
-		[Export ("delegate", ArgumentSemantic.Assign), NullAllowed]
+		[Export ("delegate", ArgumentSemantic.Weak), NullAllowed]
```

Comment:
it's never clear if such changes are (a) a fix to match the existing implementation or (b) a change in behaviour

since (b) is an issue for ObjC apps as well I tend to think it's (a) and it's fine/best to update